tests/flow/test_bruteforce.py

```diff
@@ -663,8 +663,8 @@ def range_query(self, dist_func):
         # Verify that we got exactly all vectors within the range
         results, keys = get_ground_truth_results(dist_func, self.query_data[0], label_to_vec_list, res_num)
 
-        assert_allclose(max(bf_distances[0]), results[res_num-1]["dist"], rtol=1e-05)
-        assert np.array_equal(np.array(bf_labels[0]), np.array(keys))
+        assert_allclose(bf_distances[0], [results[i]["dist"] for i in range(res_num)], rtol=1e-05)
+        assert self.compute_correct(bf_labels[0], bf_distances[0], keys, results) == res_num
```

Comment:
What's the change here exactly?

Reply:
Check that all the results are matching the expected, and not only that the last one is within the range

Reply:
Here now, we also handle ties, where the order of the labels may be different between sorting implementations (we only sort by the distance)